### PR TITLE
HKG: Car Port for Genesis GV70 2022 (3.5T, HDA2)

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**

- [x] added entry to CAR in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot:
  - `f1b6af6fdf6c0a78`: `f1b6af6fdf6c0a78/00000004--5a6db3a9bd`
  - `d81673ef19d79a4e`:
- [x] route with stock system: `afe09b9f5d3f3548/00000005--fe8cd35e57`
- [x] car harness used (if comma doesn't sell it, put N/A): `N/A` (See below)
  - Modified Harness M: https://github.com/commaai/opendbc/issues/1297

Thanks to the community 2022 Genesis GV70 3.5T owners taylababygirl (Discord) and animatronichobbes.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -576,7 +576,8 @@ class CAR(Platforms):
     "GENESIS GV70 1ST GEN",
     [
       HyundaiCarInfo("Genesis GV70 (2.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_l])),
-      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_m])),
+      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "Highway Driving Assist", car_parts=CarParts.common([CarHarness.hyundai_m])),
+      HyundaiCarInfo("Genesis GV70 (3.5T Trim) 2022-23", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_l])),
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:580]: This could make sense, but you're claiming this car is HDA 2 without the ADAS driving ECU?

Comment (sshane) [reply selfdrive/car/hyundai/values.py:580]: Doesn't look like there is HDA2 for this car. https://www.genesis.com/content/dam/genesis/us/pdf/2022/2022-Genesis%20GV70%20Brochure_vfin.pdf

Comment (sunnyhaibin) [reply selfdrive/car/hyundai/values.py:580]: @sshane According to the spec sheet from Genesis, this car (2022 GV70 3.5T AWD) comes with Highway Driving Assist II: https://www.genesis.com/content/dam/genesis/us/com/pdf/2024/2024%20GV70%20Specs%20Sheet_vfin.pdf

image

**Comment by sunnyhaibin** [comment]
Working route: `f1b6af6fdf6c0a78/00000004--5a6db3a9bd`

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (sshane) [reply selfdrive/car/hyundai/values.py]: That's the 2024 model though, check 2022 and 2023, they both say both trims just have HDA (no 2)

Comment (nastroseidon) [reply selfdrive/car/hyundai/values.py]: If it helps, here is the window sticker for my 2022 Genesis GV70 2.5t Sport prestige that has HDA2. 
IMG_1104

**Comment by sshane** [comment]
We've moved the car interfacing code to our [`opendbc`](https://github.com/commaai/opendbc) repository, which is now the new home for car ports and fingerprints. Please re-open your pull request against [`opendbc`](https://github.com/commaai/opendbc) at your convenience by using this command below. This will transform all changes under `selfdrive/car/` to `opendbc_repo/opendbc/car/`. Make sure you have initialized submodules beforehand and have checked out the latest `opendbc` commit.

```bash
PR_NUMBER=33045
curl -L https://github.com/commaai/openpilot/pull/$PR_NUMBER.patch | sed -e 's/selfdrive\/car/opendbc_repo\/opendbc\/car/g' | git apply -v --reject
```

Simply replace the PR number with your own. Once done, add the files, fix any conflicts, and open a new PR. Alternatively, you may start a new PR from scratch if that is easier for you.

**Pull request closed without merging**